assembly/assembly-che-tomcat8/src/assembly/bin/setenv.sh

```diff
@@ -14,7 +14,7 @@
 [ -z "${CHE_LOCAL_CONF_DIR}" ]  && CHE_LOCAL_CONF_DIR="${CATALINA_HOME}/conf/"
 
 #Global JAVA options
-[ -z "${JAVA_OPTS}" ]  && JAVA_OPTS="-Xms256m -Xmx2048m -Djava.security.egd=file:/dev/./urandom"
+[ -z "${JAVA_OPTS}" ]  && JAVA_OPTS="-XX:MaxRAMPercentage=85.0 -Djava.security.egd=file:/dev/./urandom"
```

Comment:
Why 85%? How much memory are we currently giving to che-server pod with `Xmx2048m` ?

Reply:
> Why 85% - 

just an assumption.

>How much memory are we currently giving to che-server pod with Xmx2048m ? - 

1G ))

Reply:
it's working because che-operator is always set JAVA_OPTS with -XX:MaxRAMFraction=2 see https://github.com/eclipse/che-operator/pull/224

Reply:
ok, so it ignores whatever we write here? What about helm ?

Reply:
all configs synced.